packages/widget/src/hooks/useSwitchEvmChainIfNeeded.tsx

```diff
@@ -1,32 +1,30 @@
-import { useCallback, useEffect, useMemo } from "react";
-import { useGetAccount } from "./useGetAccount";
+import { useCallback, useEffect } from "react";
 import { useAccount } from "wagmi";
 import { sourceAssetAtom } from "@/state/swapPage";
 import { useAtomValue } from "jotai";
 import { skipChainsAtom } from "@/state/skipClient";
+import { evmWalletAtom } from "@/state/wallets";
 
 export const useSwitchEvmChainIfNeeded = () => {
-  const getAccount = useGetAccount();
```

Comment:
`useGetAccount` is just really shitty and will cause issues when its included in a dependency array of a useEffect